Corona-Warn-App/src/main/res/layout/include_main_card.xml

```diff
@@ -42,6 +42,7 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/main_card"
         style="@style/card"
+        android:contentDescription="@string/hint_external_webpage"
```

Comment:
that's an include, that view can be reused, so this attribute should be set where the include is used but not directly on a reusable component. If it's needed exactly in this place pass it in as an attribute.